code/api/src/modules/style/model.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+module.exports = function(sequelize, DataTypes) {
+  let Style = sequelize.define('styles', {
+    name: {
+      type: DataTypes.STRING
+    }
+  })
+
+  Style.associate = function(models) {
+    // Style.hasMany(models.User) --> aren't we using UserStyle for this?
```

Comment:
UserStyle will belongto style and user. Theoretically should look like subscriptions relationships